scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -197,6 +203,25 @@ def flag_mapped_resources_archives_and_ignored_directories(self):
         flag.flag_ignored_directories(self.project)
         d2d.flag_processed_archives(self.project)
 
+    def perform_janitorial_tasks(self):
+        """
+        On deployed side
+            - PurlDB match files with ``no-java-source``, ``too-many-maps`` and empty
+                status, if no match is found update status to ``requires-review``.
+            - Update status for uninteresting files.
+
+        On devel side
+            - Update status for not deployed files.
+        """
+        d2d.match_resources_with_no_java_source(project=self.project, logger=self.log)
+        d2d.match_unmapped_resources(
+            project=self.project,
+            matched_extensions=self.purldb_resource_extensions,
+            uninteresting_extensions=self.uninteresting_extensions,
+            logger=self.log,
+        )
```

Comment:
We will have leftovers from matching that are not matched and we may also run a scan on these on the To/ side.

Reply:
leftovers will marked as ``requires-review``

Reply:
@pombredanne we want a step that runs scan on leftover `to` resources ?

Reply:
Added step to scan unmapped `to` files